pkg/util/cert/certutil.go

```diff
@@ -166,7 +149,12 @@ func GenerateJKSFromByte(certByte []byte, privateKey []byte, caCert []byte) (out
 
 // GenerateJKS creates a JKS with a random password from a client cert/key combination
 func GenerateJKS(certs []*x509.Certificate, privateKey []byte) (out, passw []byte, err error) {
-	pKeyRaw, err := DecodeKey(privateKey)
+	pKeyRaw, err := DecodePKCS1PrivateKeyBytes(privateKey)
+	if err != nil {
+		return
+	}
+
+	pKeyPKCS8, err := x509.MarshalPKCS8PrivateKey(pKeyRaw)
```

Comment:
Is this thing fails if the format is other than pkcs1? I think we should keep using the old Decode function with this.

Reply:
The prev function generates error x509.ParsePKCS1PrivateKey()  if it get other than pkcs1